src/reducers/index.js

```diff
@@ -1,3 +1,16 @@
 import { combineReducers } from 'redux';
+import { eventsReducer } from './eventsReducer/eventsReducer';
+import { userReducer } from './userReducer/userReducer';
+import { wishlistReducer } from './wishlistReducer/wishlistReducer';
+import { attendingReducer } from './attendingReducer/attendingReducer';
+import { errorReducer } from './errorReducer/errorReducer';
+import { loadingReducer } from './loadingReducer/loadingReducer';
 
-export const rootReducer = combineReducers({});
\ No newline at end of file
+export const rootReducer = combineReducers({
+  events: eventsReducer,
+  user: userReducer,
+  wishlist: wishlistReducer,
+  attending: attendingReducer,
+  loading: loadingReducer,
+  error: errorReducer,
+});
```

Comment:
Our state tree and the accompanying reducers & their tests look great. I think this will be a great starting point for our project!!